pyatlan/generator/templates/methods/attribute/a_p_i_query.jinja2

```diff
@@ -0,0 +1,38 @@
+
+        @classmethod
+        @init_guid
+        def create(
+            cls, 
+            *, 
+            name: str, 
+            connection_qualified_name: str, 
+            api_input_field_count: Optional[int] = None,
+            api_query_output_type: Optional[str] = None,
+            api_query_output_type_secondary: Optional[str] = None,
+            is_object_reference: Optional[bool] = False,
+            reference_api_object_qualified_name: Optional[str] = None,
+        ) -> APIQuery.Attributes:
```

Comment:
Should be `creator()` (ref: https://github.com/atlanhq/atlan-python/pull/403/files#r1800760792)